tests/test_verifier.py

```diff
@@ -46,6 +45,18 @@ class BounceMessageVerifierTest(TestCase):
         "UnsubscribeURL": "https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:364852123998:test-email:0fe0bffc-6470-4502-9414-d5e3d9fdd71e",
     }
 
+    valid_msg_missing_fields = {
```

Comment:
It would be good to specify what fields are missing compared to the base, but can be a follow-up

Reply:
Sure. In the forthcoming PR.